ush/ocnpost.py

```diff
@@ -0,0 +1,646 @@
+#------------------------------------------------------------------
```

Comment:
There are too many questions from this script.
1. Is this really efficient?  Is python the right choice for doing this or is a compiled language (e.g. Fortran) better suited for the operations of this size.
2. There are no tests for this.  Every python program that is added must include tests and pass standards.
3. Why are use using those `parm` files?  Can that information not be described in a better form e.g. `yaml` or `namelist` (if using Fortran).

Reply:
1.  I definitely agree that Fortran is more efficient with computation, which is why most of the intense computation is performed using a Fortran to Python interface generator (f2py) provided by Numpy. 
2. Sure, formal tests will be performed. 
3. There's no particular reason. I can change the format of the parm files to yaml.

Reply:
1. From my experience, FORTRAN is going to be the most efficient way to do this. Further, there is no need to wrap the FORTRAN with `f2py`. If you have the weights already, you should only need to loop over the coefficients to do the remapping. Using Python only complicates things and makes the application less portable.

2. A simple unit test would be using a static ESMF coefficients file, remapping from 5p0 to a comparable Gaussian grid nominal resolution and taking the difference of the fields. I would expect the different to be on the order of 0 aside from rounding error.

3. This is the perfect scenario for which to use a YAML configuration. The configuration file you provide is difficult to interpret. A YAML variation would be much more readable and easier to digest.